galleon-pack/src/main/resources/layers/standalone/microprofile-platform/layer-spec.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" ?>
```

Comment:
@rhusar please consider reading [this comment](https://docs.google.com/document/d/1FXdEQc1Q8v1GmsTbwb6XDl_GTFrcFYuQPpkAOV6hrUI/edit?disco=AAAAJxVMytU) in order to validate the location where this layer is defined

Reply:
@tommaso-borgato The comment confirms the location is correct. Resolving.